### ISSUE TITLE
Cannot find a file named '...\bin\Debug\getAttribute.js' or an embedded resource with the id 'getAttribute.js'.

### ISSUE THREAD

**Issue by InPermutation** [SeleniumHQ/selenium#3137]
## Meta -
OS:  Windows 8.1 Professional
Selenium Version:  3.0.0 (for .NET 4.0)
Browser:  Firefox
Browser Version:  50

## Expected Behavior -
`selenium-dotnet-strongnamed-3.0.0.zip` should contain the same files as `selenium-dotnet-3.0.0.zip`, but with a cryptographic signature.

## Actual Behavior -
`selenium-dotnet-strongnamed-3.0.0.zip` is missing two resources that `selenium-dotnet-3.0.0.zip` contains, `getAttribute.js` and `isDisplayed.js`.

Trying to call `element.GetAttribute("id")`, for example, will throw:

```
OpenQA.Selenium.WebDriverException
Cannot find a file named '...\bin\Debug\getAttribute.js' or an embedded resource with the id 'getAttribute.js'.
```

## Steps to reproduce -
Open the DLLs inside [`selenium-dotnet-strongnamed-3.0.0.zip`](http://selenium-release.storage.googleapis.com/3.0/selenium-dotnet-strongnamed-3.0.0.zip) and [`selenium-dotnet-strongnamed-3.0.0.zip`](http://selenium-release.storage.googleapis.com/3.0/selenium-dotnet-3.0.0.zip) using ILSpy. Expand the Resources for `WebDriver (3.0.0.0)` in each.

<img width="460" alt="ilspy" src="https://cloud.githubusercontent.com/assets/1096993/20450755/e855e38c-adc0-11e6-8cd8-f906c4453fb6.png">

**Comment by lewis-almeyda** [comment]
It's happening for me in Win10 as well.

**Comment by jimevans** [comment]
Thanks for the issue report. I'm in an airport at the moment about to board a transatlantic flight. I'll get to it at my first available opportunity.

**Comment by jimevans** [comment]
Fixed in e9433bd. Release forthcoming.

**Comment by InPermutation** [comment]
Thanks very much, @jimevans! Qunit tests are passing locally again. Now to track down that breakpoint somebody stuck in Firefox...

Happy Thanksgiving.
